electrumsv/network.py

```diff
@@ -153,7 +153,7 @@ class DisconnectSessionError(Exception):
 
     def __init__(self, reason, *, blacklist=False):
         super().__init__(reason)
-        self.blacklist = False
+        self.blacklist = blacklist
```

Comment:
We cannot enable this unless we know it is safe. It is very likely it causes further problems in unforeseen ways. IIRC Neil disabled blacklisting.

Reply:
Ok. Have put it back to how it was to be on the safe side.